src/partials/proposal-to-leave/proposal-to-leave.html

```diff
@@ -0,0 +1,107 @@
+<div class="container proposal-to-leave">
+  <h1>
+    Master the <span class="language">Language</span> with Our English Package!
+  </h1>
+  <div class="card-wrapper">
+    <div class="card outer-card">
+      <h2 class="card-title">Power of English</h2>
+      <ul class="card-content">
+        <li class="card-item">
+          <div class="icon-wrapper">
+            <svg
```

Comment:
use sprite for SVG icons

`<svg width="20" height="20" class="icon">
    <use href="#icon-check"></use>
  </svg>`